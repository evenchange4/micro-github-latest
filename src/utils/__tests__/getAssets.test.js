const getAssets = require('../getAssets');
const data = require('./data.json');

it('should return getAssets list', () => {
  expect(getAssets(data)).toMatchSnapshot();
});
