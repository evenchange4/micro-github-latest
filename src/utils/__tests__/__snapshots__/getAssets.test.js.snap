// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return getAssets list 1`] = `
Array [
  "https://github.com/evenchange4/micro-medium-api/releases/download/v1.1.0/micro-medium-api-linux",
  "https://github.com/evenchange4/micro-medium-api/releases/download/v1.1.0/micro-medium-api-macos",
  "https://github.com/evenchange4/micro-medium-api/releases/download/v1.1.0/micro-medium-api-win.exe",
]
`;
